app/assets/stylesheets/language.scss

```diff
@@ -101,3 +103,14 @@
     background-color: #f5f5f5;
   }
 }
+
+@media (max-width: 400px) {
+  .col-languages {
+    width: 100%;
+  }
+}
+
+.xs-space {
+  display: block;
+  height: 40px;
+}
```

Comment:
This was just a test, I forgot to remove before push.
I'll delete this right now and apply your suggestions. :smile: